qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -100,16 +100,20 @@ public void fillVisitNoteForm() {
 		visitNotePage.addSecondaryDiagnosis(DIAGNOSIS_SECONDARY);
 		visitNotePage.addNote("This is a visit note.");
 	}
```

Comment:
```suggestion
	}

```
Add an empty line here